docs/marketplace/dev/howtos/how-to-create-a-new-module-with-application.md

```diff
@@ -1,17 +1,25 @@
 ---
-title: "HowTo: Create a new module with application"
+title: 'HowTo: Create a new Angular Module with application'
```

Comment:
@gund, why do you have "Module" uppercase?

Reply:
As it refers to the term for consistency I've used uppercase for it.

Reply:
So, grammar-wise, the correct term, in this case, would be "Angular module".